NetSDK/Services/Cache/Classes/RedisAdapter.cs

```diff
@@ -10,7 +10,7 @@ namespace Splitio.Services.Cache.Classes
 {
     public class RedisAdapter : IRedisAdapter
     {
-        private static readonly ILog Log = LogManager.GetLogger(typeof(RedisAdapter));
+        private static readonly Logger Log = LogManager.GetLogger(typeof(RedisAdapter).ToString());
```

Comment:
@patricioe I did it this way, because if any time the class name changes, the names is automatically refreshed. If I use a constant string the name of the class will change, and the constant should be updated manually.  And yes, it is common.